Projects/Presentation/Sources/Login/LoginManager.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  LoginManager.swift
+//  Presentation
+//
+//  Created by Hohyeon Moon on 2023/05/12.
+//
+
+import Foundation
+import UIKit
+
+import AuthenticationServices
+import GoogleSignIn
+import RxMoya
+import RxSwift
+
+import Networking
+
+// MARK: - SocialLogin
+
+public enum SocialLogin {
+  case google
+  case apple
+}
+
+// MARK: - LoginManager
+
+final class LoginManager {
+  private let disposeBag = DisposeBag()
+
+  weak var loginViewController: LoginViewController?
```

Comment:
이 부분도 LoginViewController에 의존할 필요는 없을 것 같아요!
```suggestion
weak var viewController: UIViewController?
```
다른 화면에서 LoginManager를 사용해야 하는 시점이 생긴다면 문제가 발생해요